buildSrc/subprojects/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -57,7 +58,7 @@ abstract class BuildCommitDistribution : DefaultTask() {
     fun buildCommitDistribution() {
         val rootProjectDir = project.rootProject.rootDir.absolutePath
         val commit = commitBaseline.map { it.substring(it.lastIndexOf('-') + 1) }
-        tryBuildDistribution(RemoteProject.checkout(this, rootProjectDir, commit.get()))
+        tryBuildDistribution(RemoteProject.checkout(this, rootProjectDir, commit.get(), project.rootProject.buildDir.resolve("bcd")))
```

Comment:
Can we use a longer, more descriptive name? Like `commit-dist`?

Reply:
I don't know what's `bcd`, I removed it.

Reply:
I'm guessing it's from `buildCommitDistribution`.